components/core/src/package/target.rs

```diff
@@ -83,13 +83,15 @@ macro_rules! supported_package_targets {
     ) => {
         const SUPPORTED_PACKAGE_TARGETS: &'static [PackageTarget] = &[
```

Comment:
This name might need to be changed... it seems to mean "all targets we were compiled to know about", but now that probably needs clarification.